README.md

```diff
@@ -19,7 +19,7 @@
 
 # 📖 [Portfolio] <a name="about-project"></a>
 
-> This project was created to practice flexbox, Grid by building a portfolio
+> This project was created with media query and responsive web design
 
 **[Portfolio]** is a webpage with toolbar and headline section.
```

Comment:
[OPTIONAL] Kindly consider using the most recent version of [Microverse Readme template](https://github.com/microverseinc/readme-template) as it is now the recommended README for our project and it is more descriptive and easy to customize. 

[OPTIONAL] Please kindly consider fixing the README LICENSE link as it is broken. Public repositories on GitHub are often used to share open-source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software. 

You can learn more about [adding a license to a repository ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository#:~:text=In%20the%20file%20name%20field,Click%20Review%20and%20submit.)

![image](https://user-images.githubusercontent.com/78251927/206844122-29da3e28-d88c-48d3-aa52-c67b5bb8684a.png)